modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -248,7 +253,7 @@ class Session implements ISession {
 
     private int poolSize
 
-    private List<TraceObserver> observers = Collections.emptyList()
+    private List observers = Collections.emptyList()
```

Comment:
Then it would be preferable to use another typed list `List<TraceObserverV2> observersV2`